src/os/metrics/layersmetrics.js

```diff
@@ -328,8 +328,8 @@ os.metrics.LayersMetrics = function() {
     key: os.metrics.Layer.EXPORT
   });
   this.addChild(contextFeatureLayersMenuLeaf, {
-    label: 'Feature List',
-    description: 'Display a list of features in the layer.',
+    label: 'Show Features',
+    description: 'Displays metadata for features in the layer.',
```

Comment:
To me its attributes, not metadata.

Reply:
This is one of those things that will never resonate with everyone, kind of like the term "features" itself. It varies between users, and developers tend to have different opinions yet. I'm not sure what the best approach is, other than removing the "metadata for" phrase entirely.

Reply:
I think leaving the `metadata for` part out would be better. Its a very minor nit though.

Reply:
I agree, removed.